src/components/stegvisning/stegvisning-utils.ts

```diff
@@ -0,0 +1,20 @@
+import { NavigateFunction } from 'react-router-dom'
+
+import { paths, stegvisningOrder } from '@/router/constants'
+import { AppDispatch } from '@/state/store'
+import { userInputActions } from '@/state/userInput/userInputReducer'
+
+export const onStegvisningCancel = (
```

Comment:
Denne kunne vært en hook (f.eks. `useStegvisning`) som returnerte `next` og `cancel`. Da kunne den hentet `navigate` selv og slipper og må ha `useNavigate` i alle stegene.

Reply:
Kunne også tatt inn `isVeileder` for cancel

Reply:
Bra forslag! Se siste commit med opprettelse av useStegvisningNavigation.
Jeg har valgt å la `isVeileder` logikk utenfor fordi den gjelder kun noen steg i stegvisningen. Slik kan jeg enklere gjenbruke `useStegvisningNavigation.handlers.onStegvisningCancel` i `StepFeil` og `ErrorPageUnexpected` uten parameter.